Classes/Issues/IssueResult.swift

```diff
@@ -120,6 +120,29 @@ struct IssueResult: Cachable {
         )
     }
 
+    func withReviewers(
```

Comment:
Out of curiosity, why choose this pattern over using a class and mutating directly?

Reply:
Mutations = bad:

- Thread safety
- Race conditions
- Complication

Immutability requires a little bit more setup, but once the model is made you can go wild w/ it and rest assured that nothing will break.